corehq/apps/smsbillables/migrations/0015_bootstrap_icds_rates.py

```diff
@@ -1,20 +1,10 @@
 from django.db import migrations
 
-from corehq.apps.smsbillables.management.commands.bootstrap_icds_gateway import (
-    bootstrap_icds_gateway,
-)
-
-
-def create_icds_rates(apps, schema_editor):
-    bootstrap_icds_gateway(apps)
-
 
 class Migration(migrations.Migration):
 
     dependencies = [
         ('smsbillables', '0014_bootstrap_apposit_rates'),
     ]
 
-    operations = {
-        migrations.RunPython(create_icds_rates),
-    }
+    operations = {}
```

Comment:
Open to suggestions on better way to do this.

Reply:
I think this should be a list, not a dict.

Reply:
ah that was a set, updating.

Reply:
https://github.com/dimagi/commcare-hq/pull/28687/commits/e93b1b3e96f8f2c51230d9a937c0951e7f5272f1